Client/qtTeamTalk/utiltts.cpp

```diff
@@ -51,7 +51,12 @@ void addTextToSpeechMessage(const QString& msg)
     case TTSENGINE_TOLK:
 #if defined(ENABLE_TOLK)
         Tolk_PreferSAPI(ttSettings->value(SETTINGS_TTS_SAPI, SETTINGS_TTS_SAPI_DEFAULT).toBool());
-        Tolk_Output(_W(msg));
+        if (ttSettings->value(SETTINGS_TTS_OUTPUT_MODE, SETTINGS_TTS_OUTPUT_MODE_DEFAULT).toString() == "Braille")

```

Comment:
Using enum should be changed to:
`
switch(ttSettings->value(SETTINGS_TTS_OUTPUT_MODE, SETTINGS_TTS_OUTPUT_MODE_DEFAULT).toInt())
{
    case  TTS_OUTPUTMODE_BRAILLE :
        Tolk_Braille(_W(msg));
        break;
    case  TTS_OUTPUTMODE_SPEECHBRAILLE :
        Tolk_Speak(_W(msg));
        break;
....
}
`